CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   `DeviceSubType::MESSAGING_NVME_OF_NAMESPACE`.
 - Added `MemoryAttribute::SPECIAL_PURPOSE`, `MemoryAttribute::CPU_CRYPTO`,
   `MemoryAttribute::ISA_VALID`, and `MemoryAttribute::ISA_MASK`.
+- Added structs to represent each type of device path node.
```

Comment:
This could be rephrased to make it clearer that we now have support for _all of the device path node types_ defined in the current version of the UEFI spec.

Reply:
done